ch3/docker-compose.yaml

```
@@ -0,0 +1,59 @@
+version: "3.7"
+services:
+  backend-store:
+    image: postgres:14.0
+    container_name: backend-store
+    environment:
+      POSTGRES_USER: myuser
+      POSTGRES_PASSWORD: mypassword
+      POSTGRES_DB: mydatabase
+      TZ: Asia/Seoul
+    ports:
+      - 5432:5432
+    healthcheck:
+      test: ["CMD", "pg_isready", "-q", "-U", "myuser", "-d", "mydatabase"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+  artifact-store:
+    image: minio/minio
+    container_name: artifact-store
+    working_dir: /usr/app
+    ports:
+      - 9000:9000
+      - 9001:9001
+    environment:
+      MINIO_ROOT_USER: mystorage
+      MINIO_ROOT_PASSWORD: mystoragepw
+    command: server /usr/app --console-address :9001
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
+      interval: 30s
+      timeout: 20s
+      retries: 3
+
+  mlflow:
+    platform: linux/amd64
+    container_name: mlflow
+    build:
+      context: .
+      dockerfile: ./Dockerfile
```

```suggestion
    build:
      context: .
      dockerfile: ./Dockerfile
    container_name: mlflow-server
```
위의 service 들과 convention 을 동일하게 하면 좋을 것 같아요. 그리고 container name 을 mlflow 보다는 mlflow-server 가 더 좋을 것 같아요.

---

반영했습니다 :)